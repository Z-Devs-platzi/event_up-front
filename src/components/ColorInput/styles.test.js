import React from 'react';
import renderer from 'react-test-renderer';
import {
  InputSwatch,
  InputPopOver,
  InputCover,
  InputColor,
  LabelInput,
  ColorContainerInput,
} from './styles';
import 'jest-styled-components';
describe('test styled-components for the InputSwatch component', () => {
  it('should test the styled-component file', () => {
    const styles = renderer.create(<InputSwatch />).toJSON();
    expect(styles).toMatchSnapshot();
  });
  it('should test the styled-component file', () => {
    const styles = renderer.create(<InputPopOver />).toJSON();
    expect(styles).toMatchSnapshot();
  });
  it('should test the styled-component file', () => {
    const styles = renderer.create(<InputCover />).toJSON();
    expect(styles).toMatchSnapshot();
  });
  it('should test the styled-component file', () => {
    const styles = renderer.create(<InputColor />).toJSON();
    expect(styles).toMatchSnapshot();
  });
  it('should test the styled-component file', () => {
    const styles = renderer.create(<LabelInput />).toJSON();
    expect(styles).toMatchSnapshot();
  });
  it('should test the styled-component file', () => {
    const styles = renderer.create(<ColorContainerInput />).toJSON();
    expect(styles).toMatchSnapshot();
  });
});
