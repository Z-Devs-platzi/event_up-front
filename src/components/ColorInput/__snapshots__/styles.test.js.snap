// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test styled-components for the InputSwatch component should test the styled-component file 1`] = `
.c0 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  position: relative;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  background: '#fff';
  border-radius: 50%;
  border: 1px solid #808080;
  boxshadow: var(--shadow);
  cursor: pointer;
  width: 40px;
  height: 40px;
  margin: 5px;
}

<div
  className="c0"
/>
`;

exports[`test styled-components for the InputSwatch component should test the styled-component file 2`] = `
.c0 {
  position: absolute;
  top: 50px;
  left: 160px;
  z-index: 2;
}

<div
  className="c0"
/>
`;

exports[`test styled-components for the InputSwatch component should test the styled-component file 3`] = `
.c0 {
  position: fixed;
  top: 0px;
  right: 0px;
  bottom: 0px;
  left: 0px;
}

<div
  className="c0"
/>
`;

exports[`test styled-components for the InputSwatch component should test the styled-component file 4`] = `
.c0 {
  width: 32px;
  height: 32px;
  border-radius: 50%;
}

<div
  className="c0"
/>
`;

exports[`test styled-components for the InputSwatch component should test the styled-component file 5`] = `
.c0 {
  font-weight: bold;
  font-size: 16px;
  line-height: 19px;
  color: var(--grey);
}

<label
  className="c0"
/>
`;

exports[`test styled-components for the InputSwatch component should test the styled-component file 6`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  width: 168px;
  height: 40px;
}

<div
  className="c0"
/>
`;
